### DISCUSSION TITLE
Can't update Infinity via F-Droid updates tab

### DISCUSSION THREAD

**Discussion by johndoe432** [Docile-Alligator/Infinity-For-Reddit#1081] [Q&A] [answered]
Have this issue for some time. All of my F-Droid apps (no matter them being from the official repo or from the 3rd party ones) can be updated properly by searching for new versions in "Updates" tab. For some reason this doesn't work with Infinity. To update it I have to go to the "manage installed apps" tab -> Infinity -> Versions and only there I can update the app...

Does anyone know what could cause this?

**Comment by Wladefant** [comment, marked as answer]
Yes, normal issue, i had the same problem. Download the neostore (Fdroid client) instead
